doc/api/fs.md

```diff
@@ -448,10 +448,14 @@ checks fail, and does nothing otherwise.
 <!-- YAML
 added: v0.6.7
 changes:
+  - version: v8.0.0
```

Comment:
I would still prefer to stick with REPLACEME for the versions themselves (unless you’re reasonably sure that this doesn’t need to be fixed up again :smile: )